app/schema/api.py

```diff
@@ -154,7 +154,7 @@ class VlabAccountCreationIn(BaseModel):
 
     id: UUID
     name: str
-    balance: Decimal = D0
+    balance: Annotated[Decimal, Field(default=D0, ge=D0)]
```

Comment:
Minor, with the Annotated syntax the default can be provided after the equal sign
```suggestion
    balance: Annotated[Decimal, Field(ge=D0)] = D0
```